dockerfiles/hpcc/templates/dllserver-claim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dllserver-pv-claim
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 3Gi
```

Comment:
What is the long term plan for this?  Should this be a parameter configurable in the Values.yaml?

Reply:
Yes, probably.